features.md

```diff
@@ -24,6 +24,7 @@ specification.
 | `display-capture` | [Media Capture: Screen Share][media-capture-screen-share] | |
 | `document-domain` | [HTML][html] | Behind a flag in Chrome<sup>[1](#fn1)</sup> |
 | `encrypted-media` | [Encrypted Media Extensions][encrypted-media] | [Chrome 64](https://www.chromestatus.com/feature/5023919287304192) |
+| `geolocation` | https://github.com/w3c/permissions/pull/163 | [Chrome 64](https://www.chromestatus.com/feature/5023919287304192) |
```

Comment:
Can we link to the actual spec here? (Not sure if there's a good anchor nearby, but it would remove a level of indirection or two for people looking for current spec text